gnomad/utils/vep.py

```diff
@@ -802,9 +802,69 @@ def filter_vep_transcript_csqs(
     vep_root: str = "vep",
     synonymous: bool = True,
     canonical: bool = True,
-    mane_select: bool = False,
-    filter_empty_csq: bool = True,
     ensembl_only: bool = True,
+    filter_empty_csq: bool = True,
+    **kwargs,
+) -> Union[hl.Table, hl.MatrixTable]:
+    """
+    Filter VEP transcript consequences based on specified criteria, and optionally filter to variants where transcript consequences is not empty after filtering.
+
+    If `filter_empty_csq` parameter is set to True, the Table/MatrixTable is filtered
+    to variants where 'transcript_consequences' within the VEP annotation is not empty
+    after the specified filtering criteria is applied.
+
+    .. note::
+
+        By default, the Table/MatrixTable is filtered to variants where
+        'transcript_consequences' within the VEP annotation is not empty after filtering
+        to Ensembl canonical transcripts with a most severe consequence of
+        'synonymous_variant'.
```

Comment:
```suggestion
    Filter VEP transcript consequences based on specified criteria, and optionally remove variants where transcript consequences is empty after filtering.

    If `remove_empty_csq` parameter is set to True, the Table/MatrixTable is filtered
    to variants where 'transcript_consequences' within the VEP annotation is not empty
    after the specified filtering criteria is applied.

    .. note::

        By default, the Table/MatrixTable is filtered to variants where
        'transcript_consequences' within the VEP annotation is not empty after filtering
        to Ensembl canonical transcripts with a most severe consequence of
        'synonymous_variant'.
```
If the above suggestion is accepted.